common/scripted_effects/wc_artifacts_generic_scripted_effects.txt

```diff
@@ -788,138 +790,139 @@ give_artifact_random_name_effect = {
 
 		## Jewelry
 		# Amulets
-		amulet_1 = { set_name="amulet_1" }
+		amulet_1 = { event_target:target_renamed_artifact = { set_name="amulet_1" } }
```

Comment:
Just an idea, will it work if you put `trigger_switch` in `event_target:target_renamed_artifact` instead of it?

Reply:
That's good idea, thanks